modules/kargoBidAdapter.js

```diff
@@ -320,7 +325,7 @@ function getLocalStorageSafely(key) {
   }
 }
 
-function getUserIds(tdidAdapter, usp, gdpr, eids) {
+function getUserIds(tdidAdapter, usp, gdpr, eids, gpp) {
   const crb = spec._getCrb();
   const userIds = {
```

Comment:
should this be `const`? it's getting modified below

Reply:
This is fine as long as the data type doesn't change from an object. JS allow adding key value into the object otherwise.